### DISCUSSION TITLE
Hot hot hot

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#195] [Q&A]
My stepper motors on both Z and X get toasty after a while of use. So much so it melted my mounts! Ive decided to reprint them in PAHT-CF but I do wonder if maybe they are getting fed too much current. Is it worth going a step down on the dip switches for the drivers? I dont want to loose too much torque but at the same time these things are running really hot!

**Comment by kachurovskiy** [comment]
Yes, it's worth dropping the current and seeing if the reduced torque is sufficient.

Another option is switching to closed loop steppers which heat up much less.

Yet another option is to thermally sink the motors into the lathe apron/body though metal shims and such.

**Reply by Festivejelly** [reply]
I guess closed loop steppers need different drivers? Good shout with the shims, that could work for the X crosslide.

I have a bit of space around the Z motor so I suppose I could always cover it with some spare heatsinks that ive got.

I think ill drop it to the next setting down for peak current and see how that fares. Its been working well for me so far and accurate too. But I had it on quite a long time today while testing stuff and I guess it become too hot for the PLA+ mounts.

**Comment by Festivejelly** [comment]
@kachurovskiy On your hardware page you mention these closed loop steppers as an option https://www.aliexpress.com/item/1005005598264428.html

There seem to be 4 different ones there which would you say would be sufficient, and also do you have a recommendation for a closed loop one for the x Axis? I guess these would do? https://www.aliexpress.com/item/1005006019183038.html

**Reply by kachurovskiy** [reply]
For X axis you should be able to thermally sink the motor into the apron with some metal shims so hopefully temperature isn't a problem there.

I'm not sure if the integrated closed loop motor would fit into the X 3D printed mount, especially that they have wires sticking out the back and not on the side.

As a side note, more than one person reported having bad time with integrated closed loop motors - they are going into error mode for no apparent reason - I didn't see that myself though.
